src/classes/BDI_MigrationMappingHelper.cls

```diff
@@ -37,125 +37,131 @@
-                }
+            DescribeFieldResult sourceField = describeFieldResultsByFieldName.get(dataImportFieldName);
+            BDI_MigrationMappingUtility.HelpTextFieldMapping helpTextFieldMapping =
+                new BDI_MigrationMappingUtility.HelpTextFieldMapping(sourceField);
+
+            if (helpTextFieldMapping.isUsable) {
+                // Pass help text values to a new instance of MappingDetail wrapper class
+                BDI_MigrationMappingUtility.DataImportFieldMapping newDataImportFieldMapping =
+                    new BDI_MigrationMappingUtility.DataImportFieldMapping(
+                        sourceField.Label,
+                        sourceField.Name,
+                        helpTextFieldMapping.dataImportObjectName,
+                        helpTextFieldMapping.targetFieldAPIName,
+                        helpTextFieldMapping.targetObjectAPIName);
+
+                dataImportFieldMappings.add(newDataImportFieldMapping);
+            } else if (helpTextFieldMapping.isPotentialMapping) {
+                // Park suspected help text mappings
+                BDI_MigrationMappingUtility.HelpTextFieldMapping badHelpTextMapping =
+                    new BDI_MigrationMappingUtility.HelpTextFieldMapping(sourceField);
+                badHelpTextFieldMappings.add(badHelpTextMapping);
             }
         }
-
-        // Add import status and import lookup field values
-        for (String key : helpTextFieldMap.keySet()) {
-            BDI_HelpTextCMT.MappingDetail mappingDetail = helpTextFieldMap.get(key);
-            setImportStatusAndLookupFields(mappingDetail, importStatusAndLookupMap);
-        }
-
-        return helpTextFieldMap;
     }
 
-    /**
-     * @description Set import lookup and import status fields on the MappingDetail instance
-     * @param  mappingDetail Instance of MappingDetail where fields are to be set
-     * @param  importStatusAndLookupMap Map holding the field values to be set to MappingDetail
-     */
-    private void setImportStatusAndLookupFields(BDI_HelpTextCMT.MappingDetail mappingDetail,
-            Map<String, BDI_HelpTextCMT.ImportStatusAndLookupParts> importStatusAndLookupMap) {
-
-        BDI_HelpTextCMT.ImportStatusAndLookupParts importStatusAndLookupParts =
-                importStatusAndLookupMap.get(mappingDetail.dataImportObjectName);
-
-        mappingDetail.importedRecordField = importStatusAndLookupParts.importLookupFieldName;
-        mappingDetail.importedRecordStatusField = importStatusAndLookupParts.importStatusFieldName;
+    /*******************************************************************************************************
+    * @description Method to get list of good help text mappings. Used in conjuction with
+    * populateListsFromHelpTextMappings()
+    *
+    * @return List of DataImportFieldMappings
+    */
+    public List<BDI_MigrationMappingUtility.DataImportFieldMapping> getDataImportFieldMappingsFromHelpText() {
```

Comment:
This should be housed in the Utility - and maybe renamed to `getDataImportFieldMappings` since the caller doesnt know what the source was necessarily, and to be consistent with the name of the `getBadHelpTextMappings` method below.